### PR TITLE
[16.0][MIG] product_standard_margin: Migration to 16.0

### PR DESCRIPTION
**Pull request by amarcosg**

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]

File: product_standard_margin/models/product_product.py [hunk 1]
```diff
@@ -35,8 +33,8 @@ class ProductProduct(models.Model):
         compute="_compute_margin",
         string="Theorical Margin (%)",
         store=True,
-        digits=dp.get_precision("Product Price"),
-        help="Markup rate is [ Theorical Margin / sale price (Wo Tax) ] "
+        digits="Product Price",
+        help="Margin rate is [ Theorical Margin / sale price (Wo Tax) ] "
```
Comment (legalsylvain) [inline product_standard_margin/models/product_product.py:37]: Good catch !

**Review by legalsylvain — APPROVED** [review]
Diff review. Thanks !

**Review by hugo-cordoba — APPROVED** [review]
LGTM (code review)

**Comment by hugo-cordoba** [comment]
Hi @pedrobaeza, can you check this?

Regardss!!

**Pull request merged**